### PR TITLE
Unify Crash Table

### LINKED ISSUE #230: VZE: Unify Crash table on one page (consolidate QA crash table)
- [x] Update the columns in the table on `/crashes` to match the one on `/crashes/qa`. 
- [x] Delete `/crashes/qa` page
- [x] Drop "Unit Body Type" column
- [x] Update the navigation for just Crash link.

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes #230

### CODE REVIEW

File: atd-vze/src/views/Crashes/Crashes.js [hunk 1]
```diff
@@ -1,72 +1,166 @@
...
+  columns: {
+    crash_id: {
+      primary_key: true,
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Crash ID",
+      label_table: "Crash ID",
+      type: "Int",
+    },
+    case_id: {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: I don't think we need case_id as a column. Do you remember that request? I know you had it in the QA grid table but do you remember if there was a specific request to add it?

File: atd-vze/src/views/Crashes/Crashes.js [hunk 2]
```diff
@@ -1,72 +1,166 @@
...
+      label_table: "Secondary Address",
+      type: "String",
+    },
+    tot_injry_cnt: {
+      searchable: false,
+      sortable: true,
+      label_table: "Injury Count",
+      type: "Int",
+    },
+    death_cnt: {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: note here that we'll need clarification on whether to display this field or the `apd_confirmed_death_count` once that is implemented. Can you make a note to get clarification?

File: atd-vze/src/_nav.js [hunk 3]
```diff
@@ -21,15 +21,10 @@ export default {
       icon: "icon-shield",
       children: [
         {
-          name: "Crashes",
+          name: "All Crashes",
```
Comment (mateoclarke) [inline atd-vze/src/_nav.js]: Can we go one step further and remove these children all together and just make line 20 above link to the `/crashes` url?

File: atd-vze/src/_nav.js [hunk 4]
```diff
@@ -42,11 +37,6 @@ export default {
           url: "/locations",
```
Comment (mateoclarke) [inline atd-vze/src/_nav.js]: Same here for locations, drop the children and update line 37.

File: atd-vze/src/views/Crashes/Crashes.js (hunk 2 above)
Comment (mateoclarke) [reply atd-vze/src/views/Crashes/Crashes.js]: or just add a task to that issue? 
https://github.com/cityofaustin/atd-vz-data/issues/236

File: atd-vze/src/views/Crashes/Crashes.js [hunk 5]
```diff
@@ -1,72 +1,166 @@
...
+      searchable: false,
+      sortable: true,
+      label_table: "Death Count",
+      type: "Date",
+    },
+    "collision { collsn_desc } ": {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: nice that these nested queries work! But did they request this field to be added?

File: atd-vze/src/views/Crashes/Crashes.js [hunk 6]
```diff
@@ -1,72 +1,166 @@
...
+      searchable: false,
+      sortable: false,
+      label_table: "Collision Description",
+      type: "String",
+    },
+    "units { unit_description { veh_unit_desc_desc } }": {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: 👍

File: atd-vze/src/views/Crashes/Crashes.js [hunk 7]
```diff
@@ -1,72 +1,166 @@
...
+      {
+        id: "dni_non_fatal",
+        label: "Non-Fatal Injuries",
+        filter: {
+          where: [{ nonincap_injry_cnt: "_gt: 0" }],
+        },
+      },
+    ],
+  },
+  grp_geograph: {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: missing a `y` at the end of geography

File: atd-vze/src/views/Crashes/Crashes.js [hunk 8]
```diff
@@ -1,72 +1,166 @@
...
-  "death_cnt",
-];
+let crashesQuery = new gqlAbstract(queryConf);
 
-const fieldsToSearch = [
-  { rpt_street_name: "Reported Street Name" },
-  { crash_id: "Crash ID" },
-];
+let customFilters = {
+  grp_injuries: {
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: does `grp` stand for group? If it takes two extra characters to add absolute clarity about how something works, I'm down for that small sacrifice.

File: atd-vze/src/views/Crashes/Crashes.js [hunk 9]
```diff
@@ -1,72 +1,166 @@
...
+    icon: "cab",
+    label: "Deaths & Injuries",
+    filters: [
+      {
+        id: "dni_deaths",
```
Comment (mateoclarke) [inline atd-vze/src/views/Crashes/Crashes.js]: what is `dni`?

Comment (sergiogcx) [reply atd-vze/src/views/Crashes/Crashes.js]: short for "deaths n' injuries"

**Pull request merged**